Blueprint `FilesystemCustomization.MinSize` uses platform dependent `int` type, which is too small on 32bit

**Describe the bug**
The blueprint `FilesystemCustomization.MinSize` uses platform dependent `int` type. The type's size is platform-dependent and on 64bit systems it is 64 bit, but on 32bit systems, it is 32 bits only.

https://github.com/osbuild/osbuild-composer/blob/bc9e340ca53cf646c684692d221bf1a581988ee0/internal/blueprint/customizations.go#L74-L77

We build osbuild-composer for Fedora on `armv7hl`, which is 32 bits only. The latest release of osbuild-composer therefore does not build on `armv7hl`

- F33: https://koji.fedoraproject.org/koji/taskinfo?taskID=75743590
- F34: https://koji.fedoraproject.org/koji/taskinfo?taskID=75743596
- F35: (does not build because of missing dependencies due to Beta freeze)
- rawhide: https://koji.fedoraproject.org/koji/taskinfo?taskID=75743581

This produced an error while building composer, such as:
```
[thozza@armv7-test01 osbuild-composer-34][PROD]$ go build -o bin/osbuild-composer ./cmd/osbuild-composer/
# github.com/osbuild/osbuild-composer/internal/distro/rhel85
internal/distro/rhel85/distro.go:445:42: constant 2147483648 overflows int
internal/distro/rhel85/distro.go:446:14: constant 2147483648 overflows int
# github.com/osbuild/osbuild-composer/internal/distro/rhel90
internal/distro/rhel90/distro.go:470:42: constant 2147483648 overflows int
internal/distro/rhel90/distro.go:471:14: constant 2147483648 overflows int
```

**Environment**
 * OS version (`/etc/os-release` and `/etc/redhat-release`): any 32bit platform and any OS
 * osbuild-composer version (`rpm -qi osbuild-composer)`: v34

**To Reproduce**
Steps to reproduce the behavior: `make build`

**Expected behavior**
Variable types with platform-dependent size are not used to store values which don't fit to their smallest possible size.
Filesystem customizations should use `uint64` type instead of `int`. Especially when all data structures related to partitions use `uint64` for storing the size

https://github.com/osbuild/osbuild-composer/blob/bc9e340ca53cf646c684692d221bf1a581988ee0/internal/disk/disk.go#L14-L33

**Additional context**
the minimal fix to make composer build is:
```diff
diff --git a/internal/blueprint/customizations.go b/internal/blueprint/customizations.go
index 27828ed..bff24a9 100644
--- a/internal/blueprint/customizations.go
+++ b/internal/blueprint/customizations.go
@@ -73,7 +73,7 @@ type ServicesCustomization struct {

 type FilesystemCustomization struct {
        Mountpoint string `json:"mountpoint,omitempty" toml:"mountpoint,omitempty"`
-       MinSize    int    `json:"minsize,omitempty" toml:"size,omitempty"`
+       MinSize    uint64 `json:"minsize,omitempty" toml:"size,omitempty"`
 }

 type CustomizationError struct {
@@ -258,7 +258,7 @@ func (c *Customizations) GetFilesystemsMinSize() uint64 {
        if c == nil {
                return 0
        }
-       agg := 0
+       var agg uint64
        for _, m := range c.Filesystem {
                agg += m.MinSize
        }
```

--- a/internal/blueprint/customizations.go
+++ b/internal/blueprint/customizations.go
@@ -73,7 +73,7 @@ type ServicesCustomization struct {
 
 type FilesystemCustomization struct {
 	Mountpoint string `json:"mountpoint,omitempty" toml:"mountpoint,omitempty"`
-	MinSize    int    `json:"minsize,omitempty" toml:"size,omitempty"`
+	MinSize    uint64 `json:"minsize,omitempty" toml:"size,omitempty"`
 }
 
 type CustomizationError struct {
@@ -258,7 +258,7 @@ func (c *Customizations) GetFilesystemsMinSize() uint64 {
 	if c == nil {
 		return 0
 	}
-	agg := 0
+	var agg uint64
 	for _, m := range c.Filesystem {
 		agg += m.MinSize
 	}
@@ -267,7 +267,7 @@ func (c *Customizations) GetFilesystemsMinSize() uint64 {
 	if agg%512 != 0 {
 		agg = (agg/512 + 1) * 512
 	}
-	return uint64(agg)
+	return agg
 }
 
 func (c *Customizations) GetInstallationDevice() string {

--- a/internal/disk/customizations.go
+++ b/internal/disk/customizations.go
@@ -38,7 +38,7 @@ func CreatePartitionTable(
 
 	for _, m := range mountpoints {
 		if m.Mountpoint != "/" {
-			partitionSize := uint64(m.MinSize) / sectorSize
+			partitionSize := m.MinSize / sectorSize
 			partition := basePartitionTable.createPartition(m.Mountpoint, partitionSize, rng)
 			basePartitionTable.Partitions = append(basePartitionTable.Partitions, partition)
 		}

